tests_new/integration_tests/modules/s3_datasets/global_conftest.py

```diff
@@ -117,6 +117,36 @@ def session_s3_dataset1(client1, group1, org1, session_env1, session_id, testdat
             delete_s3_dataset(client1, session_env1['environmentUri'], ds)
 
 
+
+@pytest.fixture(scope='session')
+def session_s3_dataset2_with_table(client1, group1, org1, session_env1, session_id, testdata):
+    ds = None
+    try:
+        ds = create_s3_dataset(
+            client1,
+            owner='someone',
+            group=group1,
+            org_uri=org1['organizationUri'],
+            env_uri=session_env1['environmentUri'],
+            tags=[session_id],
+        )
+        creds = generate_dataset_access_token(client1, ds.datasetUri)
```

Comment:
Hear, I think it should be `json.loads(creds)`. Otherwise it's just a string

Reply:
Yes, thank youuu!!!